src/core/tactics.ml

```diff
@@ -100,9 +100,9 @@ let handle_tactic : Sig_state.t -> Proof.t -> p_tactic -> Proof.t =
       handle_refine (add_wilds pt nb)
   | P_tac_simpl         ->
       {ps with proof_goals = Goal.simpl g :: gs}
-  | P_tac_rewrite(po,t) ->
+  | P_tac_rewrite(b,po,t) ->
```

Comment:
Arrows alignment

Reply:
Hi @gabrielhdt . As already mentioned, I am not in favor of imposing arrow/parentheses alignment as it artificially creates diffs.